src/Elastic.Apm.AspNetCore/Consts.cs

```diff
@@ -0,0 +1,8 @@
+namespace Elastic.Apm.AspNetCore
+{
+	internal static class Consts
+	{
+		internal const string Sub = "sub";
```

Comment:
I would give it a more informative name. For example
```
internal static class Consts
{
    internal static class OpenIdClaimTypes
    {
        internal const string UserId = "sub";
        internal const string Email = "email";
    }
}
```

Reply:
Ok, let's do it like that.

Adapted it.